common/utils/getCommentText.js

```diff
@@ -0,0 +1,14 @@
+const { isEmptyObject } = require('./util')
+
+const getCommentText = (comments, commentType) => {
+  if (isEmptyObject(comments)) return false
+  let commentText = ''
+  comments.forEach(comment => {
```

Comment:
I think find suits the intentions here better. 
Find will bail out as soon as it finds the required element.
```
const comment = comments.find(comment => comment.commentType === commentType)
return comment ? comment.comment : ''
```